changelog.d/5386.misc

```diff
@@ -0,0 +1 @@
+Update FUNDING.yml to move liberapay from custom to official link.
```

Comment:
I'd just use the same string as #5382 to avoid having an extra line in the changelog here